### DISCUSSION TITLE
source .glb files from google drive into model viewer

### DISCUSSION THREAD

**Discussion by skimtiyaz** [google/model-viewer#3650] [Q&A]
**i am trying to find a way load a .glb file saved on google drive in model-viewer to be embeded in google sites.**

below is an example code with the file links, The "poster" here is an .png image file and the "src" is a .glb file, When running the poster is loaded but the src does not.

i have tried many different ways but i am not able to figure out...

link to stack overflow [https://stackoverflow.com/q/73190282/10065698](url)

> <div style="padding: 50px; height: 100vw; ">
    <model-viewer bounds="tight" enable-pan crossorigin="anonymous" type=""
        src="https://drive.google.com/uc?export=view&id=1LFq4gU68yvr5cFEDyeQOPa_cNwahdKta" ar
        ar-modes="scene-viewer webxr quick-look" camera-controls environment-image="neutral"
        poster="https://drive.google.com/uc?export=view&id=1ScVvlZLVq46_2gllfG_b0zwz3GPTwOvT" shadow-intensity="1" camera-orbit="-29.39deg 79.8deg auto" exposure="0.75" shadow-softness="1" auto-rotate>
        <div class="progress-bar hide" slot="progress-bar">
            <div class="update-bar"></div>
        </div>
        <button slot="ar-button" id="ar-button">
            View in your space
        </button>
        <div id="ar-prompt">
            <img src="ar_hand_prompt.png">
    </model-viewer>
</div>

**Comment by elalish** [comment]
Yeah, I really want this feature too! Unfortunately at the moment it is disallowed by Drive's CORS policy. We should talk to them about this again...

**Reply by skimtiyaz** [reply]
Any way to fix this ?

**Reply by elalish** [reply]
No, you have to serve your GLBs from something other than Drive. Pretty much anything that's actually designed for file serving (rather than sharing) will work fine for GLBs.
